src/CMakeLists.txt

```diff
@@ -769,6 +769,8 @@ list( APPEND SOURCES
       menus/FileMenus.cpp
       menus/HelpMenus.cpp
       menus/LabelMenus.cpp
+      menus/MenuHelper.cpp
+      menus/MenuHelper.h
```

Comment:
Maybe a more suggestive name?  PluginMenuBuilder?

Reply:
It used to provide macros menu helpers too. And it doesn't build menus